backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+module.exports = {
+    "env":
+    {
+        "browser": true,
+        "es2021": true
+    },
+    "extends":
+    [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides":
+    [
+        {
+            "env": {
+                "node": true
+            },
+            "files": [".eslintrc.{js,cjs}"],
+            "parserOptions": {
+                "sourceType": "script"
+            }
+        }
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions":
+    {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins":
+    [
+        "@typescript-eslint"
+    ],
+    "rules":
+    {
+        // "indent":
+        // [
+        //     "error",
+        //     "tab"
+        // ],
+        // "linebreak-style": [
+        //     "error",
+        //     "unix"
+        // ],
```

Comment:
Why is this commented ? Can it be removed if not needed?

Reply:
yep, it was deafult rules, can be safely removed